x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_lifecycle/modal.tsx

```diff
@@ -86,9 +101,13 @@ function DslModal({ closeModal, definition, updateInProgress, updateLifecycle }:
   ];
 
   const [selectedUnit, setSelectedUnit] = useState(timeUnits[0]);
-  const [retentionValue, setRetentionValue] = useState(1);
```

Comment:
Thanks for adjusting the behavior here - could we also inititialize the state with the current value?

Reply:
fixed in https://github.com/elastic/kibana/pull/210055/commits/e664261f0fc50fb88775680d3f6d6935490f5397